src/xunit.netcore.extensions/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -28,17 +28,40 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
             Debug.Assert(ctorArgs.Count() >= 2);
 
             string issue = ctorArgs.First().ToString();
-            TestPlatforms platforms = (TestPlatforms)ctorArgs.Last();
+            TestPlatforms platforms = TestPlatforms.Any;
+            TargetFrameworkMonikers frameworks = (TargetFrameworkMonikers)~0;
+            
+            foreach (object arg in ctorArgs.Skip(1)) // First argument is the issue number.
+            {
+                if (arg is TestPlatforms)
+                {
+                    platforms = (TestPlatforms)arg;
+                }
+                else
+                {
+                    frameworks = (TargetFrameworkMonikers)arg;
+                }
+            }
+        
             if ((platforms.HasFlag(TestPlatforms.FreeBSD) && RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD"))) ||
                 (platforms.HasFlag(TestPlatforms.Linux) && RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) ||
                 (platforms.HasFlag(TestPlatforms.NetBSD) && RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"))) ||
                 (platforms.HasFlag(TestPlatforms.OSX) && RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) ||
                 (platforms.HasFlag(TestPlatforms.Windows) && RuntimeInformation.IsOSPlatform(OSPlatform.Windows)))
             {
-                yield return new KeyValuePair<string, string>(XunitConstants.Category, XunitConstants.Failing);
```

Comment:
No, as it will return category nonnetcoreapptests, category nonnetfxtests, etc. For all the TargetFrameworkMonikers that are included in the if statements when a TargetFrameworkMoniker is not passed in. If we want to include category failing would be when framework == 0 and would change framework to 0 so no other flag is included. 

If we always return category failing it will always when the attribute is present and the platform matches, will skip it for all the frameworks, that is why I rather did it through the framework flags. 

I think using the 0 approach would be more readable for people in the future that get to this file to update whatever. So I will update it to follow that and return category failing when == 0 only.